src/plugins/imagery/components/ImageryView.vue

```diff
@@ -30,51 +30,116 @@
 >
     <div class="c-imagery__main-image-wrapper has-local-controls">
         <div class="h-local-controls h-local-controls--overlay-content c-local-controls--show-on-hover c-image-controls__controls">
```

Comment:
Let's move this whole div into a new ImageControls component. This should include the zoom/pan controls as well as the filters (i know that was not your code, so sorry that you're saddled with this).

When we move into the component, let's ensure that only the work that's absolutely necessary should be emitted to the parent (ImageryView) component.